http_client.rst

```diff
@@ -854,14 +854,26 @@ To leverage all these design benefits, the cURL extension is needed.
 Enabling cURL Support
 ~~~~~~~~~~~~~~~~~~~~~
 
-This component supports both the native PHP streams and cURL to make the HTTP
-requests. Although both are interchangeable and provide the same features,
-including concurrent requests, HTTP/2 is only supported when using cURL.
+This component supports the native PHP streams, `AmpHttpClient`_ and cURL to
+make the HTTP requests. Although they are interchangeable and provide the
+same features, including concurrent requests, HTTP/2 is only supported when
+using cURL or AmpHttpClient.
+
+.. note::
+
+    To use the `AmpHttpClient`_, the `amphp/http-client`_ package must be
```

Comment:
this note looks weird. It seems like AmpHttpClient refers to the Symfony class while the link goes to the source repo of the `amphp/http-client` package